graspy/match/gmp.py

```diff
@@ -280,7 +274,7 @@ def f(x):  # computing the original optimization function
                 n_iter += 1
             # end of FW optimization loop
 
-            row, col = linear_sum_assignment(
+            _, col = linear_sum_assignment(
```

Comment:
why this change?

Reply:
Row is not used in this algorithm, so I didn't store them. 
If you think we potentially need this later, I can add it back.